app/views/runs/_title.slim

```diff
@@ -72,3 +73,36 @@ h5
             span Claim
         - else
           button.btn.btn-outline-secondary disabled=true Sign In to Claim
+  .btn-group.mr-2 role='group' aria={label: 'Calls to action'}
+    - if run.belongs_to?(current_user) && run.video_url.nil? && run.highlight_suggestion.present?
+      a.btn.btn-twitch.tip-top(
+        href='#'
+        data={toggle: :modal, target: '#highlight'}
+        title='Using the timestamps in your splits, we automatically found the stream you PBed in. Click to learn more.'
+      )
+        => icon('fab', 'twitch')
+        span Auto-Highlight Possible
+- if run.belongs_to?(current_user) && run.video_url.nil? && run.highlight_suggestion.present?
```

Comment:
Maybe add in a little info bubble here about video storage durations if you don't wanna go ahead with any of the other things?